datascience/tables.py

```diff
@@ -2586,7 +2586,12 @@ def prepare_hist_with_bin_column(bin_column):
             # This code is factored as a function for clarity only.
             weight_columns = [c for c in self.labels if c != bin_column]
             bin_values = self.column(bin_column)
-            values_dict = [(w.rstrip(' count'), (bin_values, self.column(w))) for w in weight_columns]
+            print(bin_column)
```

Comment:
We shouldn't print things to stdout.  Perhaps the print statements are a remnant of debugging, and need to be deleted?